tests/trio103_104.py

```diff
@@ -0,0 +1,102 @@
+import trio
+
+try:
+    pass
+except (SyntaxError, ValueError, BaseException):
+    raise
+except (SyntaxError, ValueError, trio.Cancelled) as p:  # error
+    pass
+except (SyntaxError, ValueError):
+    raise
+except trio.Cancelled as e:
+    raise e
+except:
+    pass
```

Comment:
Should be an error here, bare `except:` is equivalent to `except BaseException:` and we're not re-raising it!

Reply:
oh, I thought bare `except:` was equivalent to `except Exception:` for some reason. Will fix!